Vagrantfile

```diff
@@ -145,8 +145,19 @@ Vagrant.configure(2) do |config|
     c.vm.synced_folder ".", "/vagrant", type: "virtualbox"
   end
 
-  config.vm.define "fedora26", primary: true do |c|
-    c.vm.box = "bento/fedora-26"
+  config.vm.define "centos6", primary: true do |c|
+    c.vm.box = "bento/centos-6.9"
+    c.vm.network :forwarded_port, guest: 22,   host: 2229,  id: "ssh", auto_correct: true
+
+    #c.vm.provision "shell", inline: $unixProvision, privileged: false
```

Comment:
Did you leave these lines in on purpose? Or should it be commented out?

Reply:
That was unintentional. I was testing provisioning and since those steps are not idempotent I comment them out.

It's fixed now.